docs-website/docs/docs/CONTRIBUTING.md

```diff
@@ -3,22 +3,33 @@ title: Contributing
 hide_title: true
```

Comment:
This should be made in CONTRIBUTING file, as this one simply mirrors it

Reply:
Do you mean I should make the changes only in the contributing file or in both? I'm assuming you mean both but I just want to make sure ahaha

Reply:
Do I need to run `yarn docs:dev` manually? or is that run as a part of the CI?  When I run it manually, it errors out

Reply:
Try `./scripts/update-docusaurus`. If doesn't work, what does it say?

Reply:
ohh I just realized it is probably is because I ran it on Windows. The error is `'.\scripts\update-docusaurus' is not recognized as an internal or external command, operable program or batch file.`